source/api/tests/test_api_integration.py

```diff
@@ -0,0 +1,380 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+# 
+# ###############################################################################
+# PURPOSE:
+#   * Integration test for project api endpoints and workflow.
+# 
+# USAGE:
+#   python3 -m venv .venv &&
```

Comment:
For the purposes of USAGE, you don't need `&&`. That only make sense if you intend people to copy and paste all these lines as a single command into their shell.

Reply:
fixed